Python/ceval.c

```diff
@@ -5199,6 +5202,17 @@ format_awaitable_error(PyTypeObject *type, int prevopcode)
     }
 }
 
+static inline void
+exit_thread(PyThreadState *tstate)
```

Comment:
Please call this something less misleading, such as `exit_thread_if_finalizing`.

Reply:
Also, this function should probably be used also in the eval loop.